firebase-perf/src/main/java/com/google/firebase/perf/application/AppStateMonitor.java

```diff
@@ -452,23 +422,13 @@ private boolean hasFrameMetricsAggregatorClass() {
     }
   }
 
-  /**
-   * An interface to be implemented by subscribers which needs to receive app state update.
-   *
-   * @hide
-   */
-  /** @hide */
+  /** An interface to be implemented by subscribers which needs to receive app state update. */
+  /** An interface to be implemented by clients which needs to receive app state update. */
```

Comment:
Remove this line since this is redundant.

Reply:
Ahh. Merge issues.

But Good catch!